web-registry/src/components/organisms/MediaForm.tsx

```diff
@@ -183,7 +185,7 @@ const MediaForm: React.FC<MediaFormProps> = ({ submit, initialValues }) => {
         onSubmit={async (values, { setSubmitting, setTouched }) => {
           setSubmitting(true);
           try {
-            await submit(values);
+            await props.submit(values);
```

Comment:
Was there an issue with destructuring `submit` directly in the props like before?

Reply:
no issue @flagrede, this was actually an oversight on my part (I accidentally deleted `submit` when just used `props` when I saw the typescript error) + my personal habits. Basically, I tend to prefer `props` for anything passed which is only used once, and destructuring for anything that's reused (or anywhere you need to set a default value).  I also think in situations where there's a few layers of prop drilling, it gives a bit of clarity to what's being called (IE this `submit` function lives outside `MediaForm`).

I don't have strong opinions here though - if folks prefer we do it all one way or all another for clarity, I'm happy to adjust